### DISCUSSION TITLE
having the same error screen since first time I checked (4 days ago)

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#2060] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/102500730/235054996-82219314-a56c-472a-bd5d-615594f55f80.png)

**Comment by TheLastBen** [comment]
sign in and go here https://console.paperspace.com/, you might find the notebooks there

**Reply by gilroff** [reply]
still no luck :(

**Reply by TheLastBen** [reply, marked as answer]
it's probably just your account, I can access it normally
